src/App.module.css

```diff
@@ -191,11 +191,12 @@
   gap: 4px;
   max-width: calc(100vw - 60px);
   margin-top: 10px;
+  flex-wrap: wrap;
+  align-items: center;
 }
 
 .timeSliderContainer {
-  width: 100%;
-  max-width: 300px;
+  width: calc(min(50vw, 300px));
```

Comment:
This was a quick fix for scaling behavior: the time slider would previously stay at the fixed 300px width, which meant that on REALLY narrow window dimensions it would extend off the screen. Because I was also changing the layout for the speed controls, I threw this in too.